### ISSUE TITLE
Do not ignore the archivesBaseName

### ISSUE THREAD

**Issue by Vampire** [rodm/gradle-teamcity-plugin#43]
Gradle supports by default an `archivesBaseName` but you ignore it and always set it to the root project name.
Please consider the `archivesBaseName`. Actually it might be enough to not set the `baseName` explicitly in `TeamCityServerPlugin:84`, afair all `Zip`-tasks should get the `archivesBaseName` or as default the project name by default already. If one want the root projects name, he could still configure it.
If you still want to use the root project name as default instead of the project name, you might still consider using the `archivesBaseName` first if it is set.

**Comment by rodm** [comment]
I've changed this, I think will affect a few projects using the plugin, but using the standard archive properties should be supported. The `agentPlugin` task now sets the `appendix` property to avoid its archive name clashing with the `serverPlugin` task's archive name when both are applied to the same project, this is for tool plugins.

**Comment by Vampire** [comment]
Is it currently even supported to apply both agent and server plugin to the same project?
When I tried this, I got an error message that this is not supported.
Unless you eventually implement #19 of course.

Btw. maybe some additional name-clash checks would be helpful?
In an intermediary state where I set the `archiveBaseName` for all projects, I ended up with the common JAR, the agent JAR and the server JAR all having the same name, so there were multiple files with the same name in the resulting ZIPs.
Or maybe just setting the `duplicatesStrategy` of the `serverPlugin` and `agentPlugin` to `fail` so that the ZIP-tasks at least fail in such a situation.
